docs/tcp/nginx.conf

```diff
@@ -1,4 +1,4 @@
-# Nginx K8s Loadbalancer Solution
+# NGINX Loadbalancer for K8s Solution
```

Comment:
I think we should use the full name "Kubernetes" here, e.g.: NGINX Loadbalancer for Kubernetes